### PR TITLE
fix: Colors in Sketch View

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we fix the colors in the Sketch View

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift [hunk 1]
```diff
@@ -122,6 +129,15 @@ final class CanvasViewController: UIViewController, UINavigationControllerDelega
...
     func configureButtons() {
+        let enabledStateIconColor = SemanticColors.Button.textInputBarItemEnabled
+        let highlightedStateIconColor = SemanticColors.Button.textInputBarItemHighlighted
+
+        let enabledStateBackgroundColor = SemanticColors.Button.backgroundInputBarItemEnabled
+        let highlightedStateBackgroundColor = SemanticColors.Button.backgroundInputBarItemHighlighted
+
+        let enabledStateBorderColor = SemanticColors.Button.borderInputBarItemEnabled
+        let highlightedStateBorderColor = SemanticColors.Button.borderInputBarItemHighlighted
+
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift:140]: Swift lint issue

Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift]: Done!

**Review by ghuMustafa — APPROVED** [review]

File: Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift [hunk 2]
```diff
@@ -145,9 +161,21 @@ final class CanvasViewController: UIViewController, UINavigationControllerDelega
...
 
         [photoButton, drawButton, emojiButton].forEach { iconButton in
-            iconButton.setIconColor(UIColor.from(scheme: .iconNormal), for: .normal)
-            iconButton.setIconColor(UIColor.from(scheme: .iconHighlighted), for: .highlighted)
-            iconButton.setIconColor(UIColor.accent(), for: .selected)
+            iconButton.layer.borderWidth = 1
+            iconButton.layer.cornerRadius = 12
+            iconButton.clipsToBounds = true
+
+            iconButton.setIconColor(enabledStateIconColor, for: .normal)
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift:168]: Why don't you want to setup style for `iconButton`  with all states?

Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift]: Done

File: Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift [hunk 3]
```diff
@@ -145,9 +161,15 @@ final class CanvasViewController: UIViewController, UINavigationControllerDelega
...
         [photoButton, drawButton, emojiButton].forEach { iconButton in
-            iconButton.setIconColor(UIColor.from(scheme: .iconNormal), for: .normal)
-            iconButton.setIconColor(UIColor.from(scheme: .iconHighlighted), for: .highlighted)
-            iconButton.setIconColor(UIColor.accent(), for: .selected)
+            iconButton.layer.cornerRadius = 12
+            iconButton.clipsToBounds = true
+
+            iconButton.applyStyle(.iconButtonStyle)
+
+            iconButton.setIconColor(highlightedStateIconColor, for: .selected)
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift:169]: Why don't move `.selected` to the `applyStyle(.iconButtonStyle)`?

File: Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift [hunk 4]
```diff
@@ -122,6 +129,15 @@ final class CanvasViewController: UIViewController, UINavigationControllerDelega
...
 
     func configureButtons() {
+        let enabledStateIconColor = SemanticColors.Button.textInputBarItemEnabled
+        let highlightedStateIconColor = SemanticColors.Button.textInputBarItemHighlighted
+
+        let enabledStateBackgroundColor = SemanticColors.Button.backgroundInputBarItemEnabled
+        let highlightedStateBackgroundColor = SemanticColors.Button.backgroundInputBarItemHighlighted
+
+        let enabledStateBorderColor = SemanticColors.Button.borderInputBarItemEnabled
+        let highlightedStateBorderColor = SemanticColors.Button.borderInputBarItemHighlighted
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift:139]: Will you remove all these colors now?

File: Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift (hunk 3 above)
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift:169]: I didn't move it because it's not something we do across many buttons throughout the app and I didn't want to introduce an optional state of colors to the style. Thoughts?

File: Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift (hunk 4 above)
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift:139]: The ones not needed, yes I will remove them.

File: Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift (hunk 3 above)
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Sketchpad/CanvasViewController.swift:169]: I ended up moving it to the applyStyle, please take a look.

File: Wire-iOS/Sources/Helpers/StylableButton.swift [hunk 5]
```diff
@@ -28,16 +28,19 @@ class StylableButton: UIButton, Stylable {
...
 
         applyStyleToNonDynamicProperties(style: style)
     }
 
     private func applyStyleToNonDynamicProperties(style: ButtonStyle) {
         setBackgroundImageColor(style.normalStateColors.background, for: .normal)
         setBackgroundImageColor(style.highlightedStateColors.background, for: .highlighted)
-
+        setBackgroundImageColor(style.selectedStateColors?.background, for: .selected)
+
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/Helpers/StylableButton.swift:40]: Swift lint issue

File: Wire-iOS/Sources/Components/Buttons/IconButton.swift [hunk 6]
```diff
@@ -19,6 +19,21 @@
 import UIKit
 import WireCommonComponents
 
+class NonLegacyIconButton: IconButton {
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/Components/Buttons/IconButton.swift:22]: Why do we need this?

Comment (agisilaos) [reply Wire-iOS/Sources/Components/Buttons/IconButton.swift:22]: We need that because otherwise the border of the buttons are going to be black like the issue with the Invite Button https://github.com/wireapp/wire-ios/pull/5941. When we finish with the accessibility project, I'm planning to clean up the IconButton as well as other classes.

File: Wire-iOS/Sources/Helpers/StylableButton.swift (hunk 5 above)
Comment (agisilaos) [reply Wire-iOS/Sources/Helpers/StylableButton.swift:40]: Done!

File: Wire-iOS/Generated/Colors+Generated.swift [hunk 7]
```diff
@@ -204,6 +214,16 @@ internal extension ColorAsset.Color {
   }
 }
 
+#if canImport(SwiftUI)
+@available(iOS 13.0, tvOS 13.0, watchOS 6.0, macOS 10.15, *)
```
Comment (KaterinaWire) [inline Wire-iOS/Generated/Colors+Generated.swift:218]: I think these changes do not belong to this PR

Comment (agisilaos) [reply Wire-iOS/Generated/Colors+Generated.swift]: Fixed it!

**Review by KaterinaWire — APPROVED** [review]
